src/user/tipsandtricks.rst

```diff
@@ -61,6 +61,15 @@ Here is how a ``.condarc`` file would look like:
 In addition to the channel priority we recommend to always install your packages inside a new environment instead the root environment from anaconda/miniconda.
 Using envs make it easier to debug problems with packages and ensure the stability of your root env.
 
+Note: in the past ``conda-forge`` used to vendorize some of ``defaults`` dependencies that were not built in our infrastructure,
```

Comment:
Can we put this in a `..note::` box for consistency?

Reply:
Done. Thanks!